react/Header/Header.demo.js

```diff
@@ -16,7 +16,8 @@ export default {
     authenticationStatus: 'authenticated',
     userName: 'Jane Smith',
     divider: false,
-    linkRenderer: dummyLinkRenderer
+    linkRenderer: dummyLinkRenderer,
+    returnUrl: 'seek-oss.github.io/seek-style-guide'
```

Comment:
Super minor nitpick, but could you make this `/jobs`, since we don't support cross-domain return URLs?